deepchem/rl/a2c.py

```diff
@@ -413,6 +413,10 @@ def _create_rollout(self, rnn_states):
                 self._create_model_inputs(self._env.state, rnn_states))
             final_value = self.discount_factor * results[
                 self._value_index].numpy()[0]
+            try:
```

Comment:
Add a comment in a future PR explaining the logic of float vs numpy array handling in try/catch